config/project/entryTypes/event--12c6ecb1-eb21-40a2-aca1-217b86047c44.yaml

```diff
@@ -0,0 +1,292 @@
+            readonly: false
+            required: true
+            size: null
+            step: null
+            tip: null
+            title: null
+            type: craft\fieldlayoutelements\entries\EntryTitleField
+            uid: b8a3d948-a454-4787-a0bb-9680311de7ab
+            userCondition: null
+            warning: null
+            width: 100
+          -
+            dateAdded: '2025-02-26T02:53:40+00:00'
+            elementCondition: null
+            fieldUid: 9b5f9007-3f61-4857-b978-fdfbc4b29dba # Event Types
+            handle: null
+            includeInCards: false
+            instructions: null
+            label: Types
+            providesThumbs: false
+            required: false
+            tip: null
+            type: craft\fieldlayoutelements\CustomField
+            uid: 2cc09c24-ea29-47ac-aa51-21495d792080
+            userCondition: null
+            warning: null
+            width: 50
+          -
+            dateAdded: '2025-02-26T02:38:49+00:00'
+            elementCondition: null
+            fieldUid: 69339afc-d4f1-440b-a7ad-7da2ccb77972 # Image
+            handle: cover
+            includeInCards: false
+            instructions: null
+            label: 'Cover Image'
+            providesThumbs: false
+            required: false
+            tip: null
+            type: craft\fieldlayoutelements\CustomField
+            uid: 04ba682b-6348-48de-833f-c03c6dd10b6b
+            userCondition: null
+            warning: null
+            width: 50
+          -
+            dateAdded: '2025-02-26T02:38:49+00:00'
+            elementCondition: null
+            fieldUid: bb0a4a47-dcbd-41a8-9bbb-82341295f5ff # Date & Time
+            handle: startDateTime
+            includeInCards: false
+            instructions: null
+            label: 'Start Date'
+            providesThumbs: false
+            required: false
+            tip: null
+            type: craft\fieldlayoutelements\CustomField
+            uid: e30e5ed4-2a93-4299-927a-7993d8dc68b3
+            userCondition: null
+            warning: null
+            width: 50
+          -
+            dateAdded: '2025-02-26T02:38:49+00:00'
+            elementCondition: null
+            fieldUid: bb0a4a47-dcbd-41a8-9bbb-82341295f5ff # Date & Time
+            handle: endDateTime
+            includeInCards: false
+            instructions: null
+            label: 'End Date'
+            providesThumbs: false
+            required: false
+            tip: null
+            type: craft\fieldlayoutelements\CustomField
+            uid: 7f13ccc6-9d8e-4cbb-b64c-7821ab39219b
+            userCondition: null
+            warning: null
+            width: 50
+          -
+            dateAdded: '2025-02-26T02:40:53+00:00'
+            elementCondition: null
+            fieldUid: afc7f0bc-1b97-4841-a1d2-5b0385a7ca35 # Plain Text
+            handle: location
+            includeInCards: false
+            instructions: null
+            label: Location
```

Comment:
Feel like we should differentiate between _Address_/_Venue_ vs “Location”. The latter feels like could be a taxonomy and allow for filtering.